tools/pipelines/build-api-markdown-documenter.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# build-benchmark-tool pipeline
+
+name: $(Build.BuildId)
+
+parameters:
+- name: releaseBuildOverride
+  displayName: Release Build (default = not released)
+  type: string
+  default: none
+  values:
+    - none
+    - release
+- name: publishOverride
+  displayName: Publish Override (default = based on branch)
+  type: string
+  default: default
+  values:
+    - default
+    - skip
+    - force
+
+trigger:
+  branches:
+    include:
+    - main
+    - next
+    - lts
+  paths:
+    include:
+    - tools/api-markdown-documenter
+    - tools/pipelines/build-api-markdown-documenter.yml
+    - tools/pipelines/templates/build-npm-package.yml
+    - tools/pipelines/templates/include-set-package-version.yml
+    - tools/pipelines/templates/include-vars.yml
+    - tools/pipelines/templates/include-publish-npm-package.yml
+    - tools/pipelines/templates/include-publish-npm-package-deployment.yml
+    - tools/pipelines/templates/include-publish-npm-package-steps.yml
+
+pr:
+  branches:
+    include:
+    - main
+    - next
+    - lts
+    - release/*
+  paths:
+    include:
+    - tools/api-markdown-documenter
+    - tools/pipelines/build-api-markdown-documenter.yml
+    - tools/pipelines/templates/build-npm-package.yml
+    - tools/pipelines/templates/include-set-package-version.yml
+    - tools/pipelines/templates/include-vars.yml
+
+extends:
+  template: templates/build-npm-package.yml
+  parameters:
+    publishOverride: ${{ parameters.publishOverride }}
+    releaseBuildOverride: ${{ parameters.releaseBuildOverride }}
+    buildDirectory: tools/api-markdown-documenter
+    tagName:
+    taskBuild: build
+    taskBuildDocs: false
+    taskLint: false
```

Comment:
Do you want to run these in CI?

Reply:
Interesting. Yes. I have updated this, but I copied this from another template. Apparently we aren't running these consistently?